app/adminPage.html

```diff
@@ -76,41 +78,40 @@ <h4 class="search-heading">Filter Results</h4>
                             <label for="endDate">End</label>
                             <input id="endDate" type="date">
                         </div>
+                    </div>
+                    <table class="table user-score-table">
+                        <thead>
+                            <tr>
+                                <th></th>
+                                <th>Name</th>
+                                <th>Email</th>
+                                <th>Score (num)</th>
+                                <th>Percentage (%)</th>
+                                <th>Time Allowed</th>
+                                <th>Time Taken</th>
+                                <th>Date of Test</th>
+                            </tr>
+                        </thead>
+                        <tbody class="score_list user-table"></tbody>
+                    </table>
                 </div>
-                <table class="table user-score-table">
-                    <thead>
-                        <tr>
-                            <th></th>
-                            <th>Name</th>
-                            <th>Email</th>
-                            <th>Score (num)</th>
-                            <th>Percentage (%)</th>
-                            <th>Time Allowed</th>
-                            <th>Time Taken</th>
-                            <th>Date of Test</th>
-                        </tr>
-                    </thead>
-                    <tbody class="score_list user-table"></tbody>
-                </table>
-            </div>
-            <form class="new-test-container" id="testForm" method="post">
-                <h2>Add new test</h2>
-                <input name="testName" type="text" id="testName" placeholder="Test name">
-                <input type="submit" class="btn primary-button new-test-button submit" value="Add new test">
-                <div id="message-target"></div>
-                <div class="buttons">
-                    <div class="messageArea">
-                        <span id="message"></span>
+                <form class="new-test-container" id="testForm" method="post">
+                    <h2>Add new test</h2>
+                    <input name="testName" type="text" id="testName" placeholder="Test name">
+                    <input type="submit" class="btn primary-button new-test-button submit" value="Add new test">
+                    <div id="message-target">
+                        <div id="inputSubmissionConfirmation" class="alert" role="alert"></div>
                     </div>
+                </form>
+                <div class="new-question-container">
+                    <h2>Add new question</h2>
+                    <h4 class="search-heading">Current Question Count: <span id="question-count"></span></h4>
+                    <a href="addQuestion.html"><button class="btn primary-button new-question-button">Add new
```

Comment:
why a button inside an a? just put the btn classes on the a and bootstrap will make it work

Reply:
your funeral